README.md

```diff
@@ -28,40 +28,49 @@ Start ganache-cli with determinstic keys:
 ganache-cli -d
 ```
 
-Start monerod for regtest:
-```
+Start monerod for regtest, this binary is in the monero bin directory:
+```bash
+cd ./monero-x86_64-linux-gnu
 ./monerod --regtest --fixed-difficulty=1 --rpc-bind-port 18081 --offline
 ```
 
-Create a wallet (will represent Bob, who owns XMR):
+Create a wallet for "Bob", who will own XMR later on:
 ```
-./monero-wallet-cli // you will be prompted to create a wallet. remember the name and optionally the password for the upcoming steps
+./monero-wallet-cli // you will be prompted to create a wallet. In the next steps, we will go with "Bob", without password. Remember the name and optionally the password for the upcoming steps
 ```
+You do not need to mine blocks, and you can exit the the wallet-cli once Bob's account has been created by typing "exit".
 
-Start monero-wallet-rpc for Bob creating a wallet (make sure `--wallet-dir` corresponds to the directory the wallet from the previous step is in:
+Start monero-wallet-rpc. At launch it asks to open a wallet, type Bob (make sure `--wallet-dir` corresponds to the directory the wallet from the previous step is in:
 ```
 ./monero-wallet-rpc  --rpc-bind-port 18083 --password "" --disable-rpc-login --wallet-dir .
 ```
 
 Open the wallet:
+```bash
```

Comment:
I'm not sure why this is needed tbh. I opened it, then when because the wallet was open before with `monero-wallet-rpc` I had to close it first, then to mine block I don't think the wallet needs to be opened with the RPC does it?

Reply:
ah, yeah this is needed since if you stop `monero-wallet-rpc` or `monero-wallet-cli` it automatically closes the wallet, and doesn't auto-load it on startup (since the wallet-dir can contain multiple wallets) and yeah to mine blocks I think it needs to be open (not 100% sure though?)